### PR TITLE
ENH: add conda-forge setup to rever's own rever.xsh

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

File: rever.xsh [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
     ('setup.py', '        version\s*=.*,', "        version='$VERSION',")
     ]
 $CHANGELOG_FILENAME = 'CHANGELOG.rst'
 $CHANGELOG_IGNORE = ['TEMPLATE.rst']
-$TAG_REMOTE = 'git@github.com:ergs/rever.git'
+$TAG_REMOTE = 'git@github.com:regro/rever.git'
+$GITHUB_ORG = 'regro'
+$GITHUB_REPO = 'rever'
+$PROJECT = 'rever'
+$GITHUB_CREDFILE = '~/.gh_cred.txt'
```
Comment (scopatz) [inline rever.xsh]: This should not be set in the rever file ever.  This is for user's to set in their environment, if needed.  There will one credential file per project so we especially don't want to set this to something that would be shared by projects.

Comment (scopatz) [reply rever.xsh]: It would be nice if we could figure out a way to warn or prevent...

Comment (scopatz) [reply rever.xsh]: Actually, I think I know how...

Comment (CJ-Wright) [reply rever.xsh]: Ah ok, I wasn't certain.

**Comment by scopatz** [comment]
Hi @CJ-Wright - sorry that there is a conflict here now with master.  But the benefit is that there is also a fix for the bug.

**Comment by CJ-Wright** [comment]
@sRebased

File: rever.xsh [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
     ('setup.py', '        version\s*=.*,', "        version='$VERSION',")
     ]
 $CHANGELOG_FILENAME = 'CHANGELOG.rst'
 $CHANGELOG_IGNORE = ['TEMPLATE.rst']
-$TAG_REMOTE = 'git@github.com:ergs/rever.git'
+$TAG_REMOTE = 'git@github.com:regro/rever.git'
+
+$GITHUB_ORG = 'regro'
+$GITHUB_REPO = 'rever'
+$GITHUB_CREDFILE = '~/.gh_cred.txt'
```
Comment (scopatz) [inline rever.xsh]: This should still be removed...

Comment (CJ-Wright) [reply rever.xsh]: Terribly sorry, I think I got it this time.

**Comment by scopatz** [comment]
Sweet it finally passed.

**Comment by CJ-Wright** [comment]
Thank you @scopatz

**Pull request merged**